unity-renderer/Assets/DCLServices/EmotesCatalog/EmotesCatalogService.cs

```diff
@@ -50,10 +51,8 @@ private async UniTaskVoid InitializeAsyncEmbeddedEmotes()
             DisposeCT();
             InitializeAsyncEmbeddedEmotes();
         }
-        finally
-        {
-            EmbedEmotes();
-        }
+
+        EmbedEmotes();
```

Comment:
I think `EmbedEmotes()` should be moved after `embeddedEmotesSO`, otherwise, it will be called several times, won't it? I know it is not your code but can we improve it?

Reply:
sorry, it is already merged.
you should add an issue to the explorer board for that, also I noticed that there is no test coded for the "retry" logic, so it might be worth to add that coverage to the issue

Reply:
Moved to #4486